coverage.sh

```diff
@@ -4,14 +4,15 @@ set -ex
 
 TEST_CMD=${TEST_CMD:-'echo "TEST_CMD unset; assuming: make test"; make test'}
 
-COVERAGE_DIR=.coverage
+COVERAGE_DIR=$(mktemp -d)
 rm -rf $COVERAGE_DIR
 mkdir -p $COVERAGE_DIR
-pushd $COVERAGE_DIR
 if [ -z "$KEEP" ]; then trap "popd; rm -rf $COVERAGE_DIR" EXIT; fi
 
 # copy over everything
-$(which cp) -r ../* .
+$(which cp) -r ./* $COVERAGE_DIR
```

Comment:
The `which cp` seems strange to me. Why not use `/bin/cp` or simply `cp`? I don't understand why it is here.

Reply:
I have no idea to be honest, I also found this surprising.